tests/regression/01-cpa/06-pointers.c

```diff
@@ -1,3 +1,5 @@
+// PARAM: --set ana.activated "['base' ,'mallocWrapper']"
```

Comment:
The regression test for `01/06-pointers` fails because `escape` analysis is deactivated. I suppose you meant to have `--sets ana.activated[-] octApron` here as well?

Reply:
But isn't it the same if I write:
// PARAM: --set ana.activated "['base' ,'mallocWrapper']" --set ana.activated[-] "'octApron'"
because 
// PARAM: --set ana.activated "['base' ,'mallocWrapper']" 
already means that activated analysis are only base and mallocWraper?

Reply:
I mean you don't need the `--set ana.activated "['base' ,'mallocWrapper']"` part at all. At least previously it worked fine with mutex, escape, etc defaults being activated. So if it fails just because of octApron right now, you can use your added syntax to deactivate it without having an incomplete analysis list which breaks the test.

Reply:
It works fine with defaults, regardless of the octApron, so I removed the params.